src/k8s/pkg/k8sd/features/metallb/loadbalancer.go

```diff
@@ -10,18 +10,62 @@ import (
 	"github.com/canonical/k8s/pkg/utils/control"
 )
 
-func ApplyLoadBalancer(ctx context.Context, snap snap.Snap, loadbalancer types.LoadBalancer, network types.Network, _ types.Annotations) error {
+const (
+	enabledMsgTmpl      = "enabled, %s mode"
+	disabledMsg         = "disabled"
+	deleteFailedMsgTmpl = "Failed to delete MetalLB, the error was: %v"
+	deployFailedMsgTmpl = "Failed to deploy MetalLB, the error was: %v"
```

Comment:
avoid mixing case, make all of these lowercase. also elsewhere.

also, these are fine to have a few times in the code below. reading `fmt.Sprintf(deleteFailedMsgTmpl, err)` does not make it easy to track where errors are coming from.

Reply:
I was trying to achieve the exact details in the proposal:
https://github.com/canonical/k8s-snap/blob/main/docs/proposals/001-k8s-status.md#:~:text=%22Failed%20to%20deploy%20Cilium%2C%20the%20error%20was%20%25w%22

Reply:
about the number of repetitions, I feel like having things like `deleteFailedMsgTmpl` does not hurt tracking down errors. because if I duplicate the error message in a file, not only it makes it so that changing that error is a bit harder, and they might diverge (unwantedly), when you look up for error you're faced with more than one result. I think if does more harm than good.